src/textarea/textarea.js

```diff
@@ -19,7 +19,7 @@ class Textarea extends React.Component<TextareaPropsT> {
     autoFocus: false,
     disabled: false,
     error: false,
-    inputRef: React.createRef(),
+    inputRef: (React.createRef(): {current: null | HTMLInputElement}),
```

Comment:
We can probably replace all the above `null || HTMLInputElement` with `?HTMLInputElement`